sysmonconfig-trace.xml

```diff
@@ -170,6 +174,8 @@
 				<TargetImage condition="end with">\mmc.exe</TargetImage>
 				<TargetImage condition="end with">\Microsoft.Tri.Sensor.exe</TargetImage> <!-- Microsoft Defender for Identity Sensor -->
 				<TargetImage condition="end with">\Microsoft.Tri.Sensor.Updater.exe</TargetImage> <!-- Microsoft Defender for Identity Sensor -->
+				<SourceImage condition="end with">\AppData\Local\Programs\Microsoft VS Code\Code.exe</SourceImage>
+				<TargetImage condition="end with">\AppData\Local\Programs\Microsoft VS Code\Code.exe</TargetImage>
```

Comment:
```suggestion
```
I would not exclude vs code as it is an application which regularly is the source of FPs and the trace config is used only for rule development. So I think we should be able to see it at that stage.

Reply:
On the other hand it is only for process access, which normally is not an issue with vs code. I just raised the question. If you say it is too noisy, just decline the suggestion and I will merge.

Reply:
It generates a lot of FP for no reason and it's already excluded if it's installed as admin in the trace. See line L147 and L155

```yml
<TargetImage condition="is">C:\Program Files\Microsoft VS Code\Code.exe</TargetImage>
```

Reply:
Just as a side note on my machine, I have 1.3M events from sysmon and 900K are "process access" events and a lot of them are VsCode. That's why I thought I reduce it.